libs/tracing/Tracing.mli

```diff
@@ -6,22 +6,8 @@
 (* Functions to instrument the code *)
 (*****************************************************************************)
 
-val with_span :
```

Comment:
we might want to keep that function; you never know, sometimes the ppx is not appropriate.

Reply:
Fair enough